lib/attributes_changed_by_write.rb

```diff
@@ -0,0 +1,28 @@
+module ArTransactionChanges
+  module AttributeChangedByWrite
+    def attributes_changed_by_write
+      @attributes_changed_by_write ||= ActiveSupport::HashWithIndifferentAccess.new
```

Comment:
Continuing the discussion from https://github.com/dylanahsmith/ar_transaction_changes/pull/10#discussion_r72691942 we should just use `@transaction_changed_attributes` and just change the way it is being set to capture all attributes.

You said doing that causes the double save test to fail, but I tried doing that and the tests pass.  See https://github.com/dylanahsmith/ar_transaction_changes/commit/4888ac82dcce02b9428aa66134ca1dd1feab982a for reference